tracer/test/Datadog.Trace.Tests/TraceProcessors/NormalizerTraceProcessorTests.cs

```diff
@@ -0,0 +1,111 @@
+// <copyright file="NormalizerTraceProcessorTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Datadog.Trace.Tests.TraceProcessors
+{
+    public class NormalizerTraceProcessorTests
+    {
+        // https://github.com/DataDog/datadog-agent/blob/eac2327c5574da7f225f9ef0f89eaeb05ed10382/pkg/trace/traceutil/normalize_test.go#L17
+        public static IEnumerable<object[]> GetNormalizeTagValues()
+        {
+            yield return new object[] { "#test_starting_hash", "test_starting_hash" };
+            yield return new object[] { "TestCAPSandSuch", "testcapsandsuch" };
+            yield return new object[] { "Test Conversion Of Weird !@#$%^&**() Characters", "test_conversion_of_weird_characters" };
```

Comment:
You've triggered a new build since then, so you can't see the code coverage until the new one completely, but I think it would be good to check the code coverage for the `TraceUtil` class and make sure you've covered all the branches, given there's a lot of complexity in there! 🙂

Reply:
I'll check, the tests are the same as the ones in the agent.